winit/src/application.rs

```diff
@@ -615,12 +615,25 @@ pub fn run_command<A, E>(
                 }
             },
             command::Action::Window(action) => match action {
+                window::Action::Drag => {
+                    let _res = window.drag_window();
+                }
                 window::Action::Resize { width, height } => {
                     window.set_inner_size(winit::dpi::LogicalSize {
                         width,
                         height,
                     });
                 }
+                window::Action::ResizeDrag => {
+                    // TODO: Requires patch to winit
+                    // let _res = window.drag_resize_window();
+                }
+                window::Action::Maximize => {
+                    window.set_maximized(!window.is_maximized());
+                }
```

Comment:
Nit: I think this should this action should be called `ToggleMaximized`. Maybe we also have a `Action::Maximize(bool)` as well?

Reply:
Sure